networking/openshift-sdn/about-openshift-sdn.adoc

```diff
@@ -26,3 +27,9 @@ cluster and receive network traffic from those Pods and Services.
 * The _subnet_ mode provides a flat Pod network where every Pod can
 communicate with every other Pod and Service. The network policy mode provides
 the same functionality as the subnet mode.
+endif::[]
+ifdef::openshift-dedicated[]
+{product-title} SDN supports only the _network policy_ mode, which allows project
```

Comment:
@sheriff-rh, I don't think there's an OKD SDN, so using `{product-title}` token here doesn't make sense. OKD would use the same Pod network provider as OCP.

Reply:
@jboxman This is OSD (openshift dedicated), which _should_ have a `{product-title}`

Reply:
Eh, it should be "OpenShift SDN." I'll open a PR to fix it.

Reply:
https://github.com/openshift/openshift-docs/pull/18415